resources/assets/components/App.js

```diff
@@ -45,6 +50,29 @@ const App = ({ store, history }) => {
         />
       ) : null}
       <ApolloProvider client={graphqlClient(env('GRAPHQL_URL'))}>
+        {!featureFlag('sitewide_nps_survey') &&
```

Comment:
This could just be flipped to 
```suggestion
        {featureFlag('sitewide_nps_survey') &&
```

Reply:
yea but that doesn't work either. The survey doesn't appear when I do this.

Reply:
Hmm 🤔  do you have the `DS_ENABLE_SITEWIDE_NPS_SURVEY=true` set in your `.env`?

Reply:
you're right that was the one thing that I forgot to do 😓thanks so much @mendelB 🙌🏽alrighty it's working now and ready to be merged 😃